src/test/java/COSE/RegressionTest.java

```diff
@@ -1238,7 +1246,7 @@ int BuildSignedMessage(CBORObject cnControl) throws CoseException
             }
 
             if (cnSign.ContainsKey("countersign0")) {
-                AddCounterSignature0(hSignObj, cnSigners.get("countersign0"));
+                AddCounterSignature0(hSignObj, cnSign.get("countersign0"));
```

Comment:
I'm definitely not sure, but it looks to me like this should be the other way around: that the `if` condition should change to `cnSigners` instead, because there's already an `if` statement like this for `cnSign` [on line 1217 above](https://github.com/cose-wg/COSE-JAVA/pull/107/commits/59db6fea102605ef8cb9f5aa9cb269688eccba0a?diff=unified&w=1#diff-31a8d66d24177707c6d3951c99093230507994d27d2dbeeadaec3a4699f3764eL1209).

Reply:
Yeah, probably, but it doesn't change anything in the tests.

Reply:
Looks like this was changed in https://github.com/cose-wg/COSE-JAVA/pull/107/commits/425c5dfa548baf67cfbbe5bfb7c82e56fe3149dc in the end?

Reply:
I did change it, because it seems logical to me.